README.md

```diff
@@ -1,6 +1,30 @@
-# City of Helsinki Ahti project
+# City of Helsinki Ahti project 
+## available at https://ahti.app/
 
-# This repository contains the first Proof of Concept implementation by Futurice
+## This repository contains the first Proof of Concept implementation by Futurice
+## Documentation last updated 13-01-2020
+
+## General information
+
+https://drive.google.com/open?id=1Rghr1WjKDJs0Vmbkmp1yLyF6IxmX5lGn
+
+## Design sources
+Are in Figma https://www.figma.com/file/ZKAH36THbAPVs9gNdmH8N6Dh/Ahti?node-id=0%3A1
+
+## Map
+The map we use is Mapbox:
+- minimal style: `mapbox://styles/strawshield/cjxx7z1sf04rm1dl7bjryf4xf` designed by Olli Kilpi
```

Comment:
This style is no longer in use as we do not want to be dependent on Mapbox' paid services. Instead we use tiles provided by City of Helsinki (https://gitlab.com/City-of-Helsinki/ohjelmistokehitys/-/wikis/maps) and styles based on OSM-liberty (https://github.com/maputnik/osm-liberty). The map style is still WIP and needs design.

Reply:
Thanks a for the clarification, @ahie ! Can we remove it for now? The styles are stil a subject for a change in this project :sigh

Reply:
Could remove the whole Map section for now, yes.